app/script.rb

```diff
@@ -0,0 +1,184 @@
+require 'pry'
+require 'watir'
+require 'nokogiri'
+require 'json'
+require 'date'
+
+class VB_WebBanking
+  BASE_URL = "https://web.vb24.md/wb/".freeze
+  ACCOUNTS_URL = "#{BASE_URL}#menu/MAIN_215.NEW_CARDS_ACCOUNTS".freeze
+  TRANSACTIONS_URL = "#{BASE_URL}#menu/MAIN_215.CP_HISTORY".freeze
+
+  attr_reader :accounts, :transactions
+
+  def run
+    browser.goto(BASE_URL)
+    sleep 2
+    authentication_check
+    check_accounts
+    check_transactions
+    store
```

Comment:
Don't force the script to store the data into a file. Call this function on an instance when you want. Like you do with `parser.run`